luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm

```diff
@@ -7,39 +7,32 @@ <h3>Freifunk-Assistent</h3>
 
   <div class="info alert-message">
     <p class="success">
-<%= translate("Als naechstes kommen die Einstellungen für das Freifunk-Netzwerk. \
-Halte die IPs bereit, die du ueber den Wizard von config.berlin.freifunk.net \
-bekommen hast.") %>
+<%= translate("Next we set the options for the Freifunk network. Keep the IPs ready that you got from the config.berlin.freifunk.net wizard.") %>
     </p>
   </div>
 
   <div class="tile">
     <div style="text-align: center; margin: 10px">
       <a class="cbi-button cbi-button-save" href="<%=luci.dispatcher.build_url("admin/freifunk/assistent/optionalConfigs")%>?sharenet=0">
-        <%= translate("Am Freifunk-Netz teilnehmen") %>
+        <%= translate("Participate in the Freifunk-Network") %>
       </a>
     </div>
     <div class="alert-message">
       <p class="info">
-        <%= translate("Erweitere das Freifunk-Netz. Du benoetigst dazu eine \
-        WLAN-Verbindung zu einem anderen Freifunk-Router. \
-        Du kannst auf der Freifunk-Karte \
-        nachschauen, wer in deiner Naehe ist, und sie kontaktieren, \
-        damit ihr die Verbindung optimal ausrichten koennt.") %>
+        <%= translate("Expand the Freifunk-Network. You need a WiFi-connection to another Freifunk router for that. You can check whom is in your neighbourhood on the Freifunk map. Then you can contact them, to adjust the mesh connection to optimum.") %>
```

Comment:
Can you please add line breaks to keep the source code a reasonable number of chars per line?